src/devices/GrovePi/Sensors/Relay.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Device.GrovePiDevice.Models;
+using System.Device.Gpio;
+
+namespace Iot.Device.GrovePiDevice.Sensors
+{
+    public class Relay : DigitalOutput
+    {
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="grovePi">The GoPiGo3 class</param>
+        /// <param name="port">The grove Port, need to be in the list of SupportedPorts</param>
+        public Relay(GrovePi grovePi, GrovePort port) : this(grovePi, port, false)
+        { }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="goPiGo">The GoPiGo3 class</param>
+        /// <param name="port">The grove Port, need to be in the list of SupportedPorts</param>
+        /// <param name="inverted">If inverted, the relay is on when output is low and off when output is high</param>
+        public Relay(GrovePi grovePi, GrovePort port, bool inverted) : base(grovePi, port)
+        {
+            IsInverted = inverted;
+        }
+
+        /// <summary>
+        /// Switch on the relay
+        /// </summary>
+        public void On()
+        {
+            Value = PinValue.Low;
+        }
+
+        /// <summary>
+        /// Switch off the relay
+        /// </summary>
+        public void Off()
+        {
+            Value = PinValue.High;
+        }
+
+        public bool IsInverted { get; set; }
+
+        /// <summary>
+        /// Get/set the state of the relay, 0 for off, 1 for on. Wehn set, anything not 0 will be considered as on
+        /// </summary>
+        public new PinValue Value
+        {
+            get { return _value; }
+            set
+            {
+                _value = value;
+                _grovePi.DigitalWrite(Port, ((_value == PinValue.High) && !IsInverted) ? PinValue.High : PinValue.Low);
```

Comment:
If we're exposing the raw value we should also set the raw value. This probably isn't even needed- you can infer the `PinValue` if you need to from `IsOn/IsEnabled` and `IsInverted`. I presume needing to know this would be very rare and having just one get/set would make it easier to use this type correctly.

Reply:
I've added an IsOn property. Will keep this Value for consistency with the rest of the sensors.